tools/src/main/python/dlpx/virtualization/_internal/validation_schemas/plugin_importer.yaml

```diff
@@ -75,7 +75,6 @@ EXPECTED_STAGED_ARGS_BY_OP:
     initialize_impl:
```

Comment:
To make sure this validation is tested, could you please add the initialize decorator to tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/direct and tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/staged. These fake plugins are used in test_plugin_importer.py tests. One e.g. of existing test - ```    @staticmethod
    @pytest.mark.parametrize('entry_point,plugin_type',
                             [('successful:staged', 'STAGED'),
                              ('successful:direct', 'DIRECT')])
    @mock.patch('dlpx.virtualization._internal.file_util.get_src_dir_path')
    def test_successful_validation(mock_file_util, plugin_config_file,
                                   fake_src_dir):
        mock_file_util.return_value = fake_src_dir
        importer = get_plugin_importer(plugin_config_file)
        importer.validate_plugin_module()```
So, once you add the decorator and right code, initialize will also get validated.

Reply:
Thanks, I didn't know about this. Added new initialize functions, following the existing configure examples.